### DISCUSSION TITLE
2023년 03월 25일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#19] [General]
# 2023년 03월 25일

[ 2023-03-01 ] [ 2023-03-02 ] [ 2023-03-05 ] [ 2023-03-06 ] [ 2023-03-07 ] [ 2023-03-08 ] [ 2023-03-09 ] [ 2023-03-10 ] [ 2023-03-12 ] [ 20…

https://cyma-s.github.io/2023-03-25/

**Comment by drunkenhw** [comment]
오우 느끼는 것이 되게 오글거리네요~ 그냥 아무 생각없이 사는거 추천드려요~ 저는 아무 생각 없거든요~~ 대충 대충~~~ 댓글도 대충 단답니다

**Reply by Cyma-s** [reply]
그럼 박스터는 이제 TIL 출입 금지입니다 ~~~~

**Reply by drunkenhw** [reply]
죄송합니다

**Reply by drunkenhw** [reply]
어? 이거 제가 단 댓글이 아닌데요?? 삭제 부탁드려요.. 
배로 언제든지 화이팅 입니다 항상 응원하고 있어요~

**Comment by Choi-JJunho** [comment]
저는 다 내려놓고 놀기만 하는 시간을 가져보면 그때 목표를 얻을 수 있는 경우가 있었어요~
베로에게도 그런 시간이 있었으면 좋겠네요 🔥

**Reply by Cyma-s** [reply]
역시 주노밖에 없네~

**Comment by ezzanzzan** [comment]
나중에 생길 목표를 위해서 열심히 살자! 라는 목표를 두는 건 어떨까요~
물론 힘들 때면 쉬어가는 것도 좋다고 생각합니다 ˙ᵕ˙ 
베로 항상 행복했으면 좋겠어요 🥺♥️

**Reply by Cyma-s** [reply]
나중에 생길 목표를 발목 잡지 않도록 열심히 살아보는 거 좋네요~
저도 토리가 행복했으면 좋겠어요 🥰
